filament/backend/src/vulkan/VulkanFboCache.cpp

```diff
@@ -108,7 +108,7 @@ VkRenderPass VulkanFboCache::getRenderPass(RenderPassKey config) noexcept {
     VkAttachmentDescription colorAttachment {
         .format = config.colorFormat,
         .samples = VK_SAMPLE_COUNT_1_BIT,
-        .loadOp = (config.flags.clear & TargetBufferFlags::COLOR) ?
+        .loadOp = (config.flags.clear & (uint8_t)TargetBufferFlags::COLOR) ?
```

Comment:
Why not use `any` here?

Reply:
ah this is because `config.flags.clear` is not a `TargetBufferFlags`, it's just a uint8_t.

and the reason it's not a `TargetBufferFlags` is because it has an extra flag, unrelated to target buffers.

Reply:
👍 thanks for the explanation